pkgs/development/python-modules/flask-security-too/default.nix

```diff
@@ -71,6 +72,7 @@ buildPythonPackage rec {
     flask-wtf
     itsdangerous
     passlib
+    importlib-resources
   ];
```

Comment:
Update the depdendencies. We don't need the packages that flask already provides.

https://github.com/Flask-Middleware/flask-security/blob/5.3.0/pyproject.toml#L41-L51